lego/apps/events/serializers/events.py

```diff
@@ -96,6 +96,14 @@ def get_price(self, obj):
             return obj.get_price(user=request.user)
 
 
+class EventReadAuthUserDetailedSerializer(EventReadUserDetailedSerializer):
```

Comment:
Nitpicking, but maybe PublicEventSerializer and PublicEventDetailedSerializer could be more descriptive, instead of adding auth in these?

Reply:
Hmm..

We have the following serializers:
`EventRead` for list,
`EventReadDetailed` for read, but no user info or registrations
`EventReadUserDetailed` for read and user info, but no registrations
`EventReadAuthUserDetailed` for read, user info and registrations.

Maybe rename to List, Read, ReadUser and ReadDetailed?